qoe-metrics-dashboard/src/app/app.tsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
-import { Outlet } from 'react-router-dom';
+import React, { useEffect } from 'react';
```

Comment:
React wird nicht genutzt

```suggestion
import { useEffect } from 'react';
```